packages/client/src/features/context-menu/selection-service-aware-context-menu-mouse-listener.ts

```diff
@@ -33,19 +34,49 @@ export class SelectionServiceAwareContextMenuMouseListener extends MouseListener
     @inject(TYPES.IContextMenuServiceProvider) @optional() protected readonly contextMenuService: IContextMenuServiceProvider;
     @inject(TYPES.IContextMenuProviderRegistry) @optional() protected readonly menuProvider: ContextMenuProviderRegistry;
     @inject(GLSP_TYPES.SelectionService) protected selectionService: SelectionService;
+    @inject(TYPES.ViewerOptions) protected options: ViewerOptions;
```

Comment:
I don't think this is used anywhere and can be safely removed.

Reply:
Right, nice catch! Thanks!